### ISSUE TITLE
Ability to add chrome history sources?

### ISSUE THREAD

**Issue by ReneFroger** [kawabata/helm-chrome#9]
Sometimes I'm looking into Chrome history source of visited sites in the
past. As far I know, Chrome have a sort of same source for visited sites as
bookmarks. So I'm wondering if it would be possible to have the ability to
include the history source in `helm-chrome`, next to bookmarks. 

I can imagine not everyone want that, so it could be an optin with `(setq
helm-chrome-include-history t)` or something?

Is that perhaps possible?

**Comment by iNode** [comment]
It's possible in theory. But AFAIK chrome keeps it's history in sqlite DB which is locked while google-chrome is running so can't be opened even for read. How useful it would be to have this feature available only if chrome isn't running?

**Comment by ReneFroger** [comment]
sqlite DB? I thought it was deprecated, see also [this link](http://programmers.stackexchange.com/questions/220254/why-is-web-sql-database-deprecated).

**Comment by kawabata** [comment]
Indeed, I wanted that feature too ;-> , so I checked the file format and it seems
sqlite3 format, and I gave it up.  I think the situation hasn't changed yet, as my machine's
Chrome/Default/History file is sqlite3 format now.
